examples/read_binary.py

```diff
@@ -13,279 +13,106 @@
+    #       'key_type_name': 'SQL_PUBLIC_CITY_081f37cc8ac72b10f08ab1273b744497_KEY',
+    #       'query_fields': [{'default_value': None,
+    #                         'is_key_field': True,
+    #                         'is_notnull_constraint_field': False,
+    #                         'name': 'ID',
+    #                         'precision': -1,
+    #                         'scale': -1,
+    #                         'type_name': 'java.lang.Integer'},
+    #                        {'default_value': None,
+    #                         'is_key_field': False,
+    #                         'is_notnull_constraint_field': False,
+    #                         'name': 'NAME',
+    #                         'precision': 35,
+    #                         'scale': -1,
+    #                         'type_name': 'java.lang.String'},
+    #                        {'default_value': None,
+    #                         'is_key_field': True,
+    #                         'is_notnull_constraint_field': False,
+    #                         'name': 'COUNTRYCODE',
+    #                         'precision': 3,
+    #                         'scale': -1,
+    #                         'type_name': 'java.lang.String'},
+    #                        {'default_value': None,
+    #                         'is_key_field': False,
+    #                         'is_notnull_constraint_field': False,
+    #                         'name': 'DISTRICT',
+    #                         'precision': 20,
+    #                         'scale': -1,
+    #                         'type_name': 'java.lang.String'},
+    #                        {'default_value': None,
+    #                         'is_key_field': False,
+    #                         'is_notnull_constraint_field': False,
+    #                         'name': 'POPULATION',
+    #                         'precision': -1,
+    #                         'scale': -1,
+    #                         'type_name': 'java.lang.Integer'}],
+    #       'query_indexes': [],
+    #       'table_name': 'CITY',
+    #       'value_field_name': None,
+    #       'value_type_name': 'SQL_PUBLIC_CITY_081f37cc8ac72b10f08ab1273b744497'}]
 
     with city_cache.scan() as cursor:
-        print(next(cursor))
-    # (
-    #     SQL_PUBLIC_CITY_6fe650e1_700f_4e74_867d_58f52f433c43_KEY(
-    #         ID=1890,
-    #         COUNTRYCODE='CHN',
-    #         version=1
-    #     ),
-    #     SQL_PUBLIC_CITY_6fe650e1_700f_4e74_867d_58f52f433c43(
-    #         NAME='Shanghai',
-    #         DISTRICT='Shanghai',
-    #         POPULATION=9696300,
-    #         version=1
-    #     )
-    # )
+        pprint(next(cursor))
+    # (SQL_PUBLIC_CITY_081f37cc8ac72b10f08ab1273b744497_KEY(ID=1890, COUNTRYCODE='CHN', version=1),
+    #  SQL_PUBLIC_CITY_081f37cc8ac72b10f08ab1273b744497(NAME='Shanghai', DISTRICT='Shanghai', POPULATION=9696300,
+    #                                                   version=1))
```

Comment:
I'm not sure this is more readable really

Reply:
fixed